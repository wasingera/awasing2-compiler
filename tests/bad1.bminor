"multiline strings
are not part of the bminor standard"

" integer: return 0;
