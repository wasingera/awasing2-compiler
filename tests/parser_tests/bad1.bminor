x: integer = {1,};

f: function = {1, 2,3,4};
