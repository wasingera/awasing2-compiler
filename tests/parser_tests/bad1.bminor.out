parse error: syntax error
Parse failed: }
